Gopkg.toml

```diff
@@ -60,9 +60,9 @@ required = [
   name = "github.com/jcrossley3/manifestival"
   branch = "client-go"
 
-[[constraint]]
+[[override]]
   name = "knative.dev/pkg"
-  branch = "master"
+  branch = "release-0.11"
```

Comment:
Should this change be made in the release-0.11 branch of serving-operator? Or is our intention for the serving-operator master branch to be using the latest release of knative.dev/pkg moving forward?

Reply:
@garron I will kick it back to master, when we get the 0.11 out.